zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/FastLeaderElection.java

```diff
@@ -1023,9 +1016,9 @@ else if (validVoter(n.sid) && validVoter(n.leader)) {
                              */
                             if (n == null) {
                                 setPeerState(proposedLeader, voteSet);
-
                                 Vote endVote = new Vote(proposedLeader,
-                                        proposedZxid, proposedEpoch);
+                                        proposedZxid, logicalclock.get(), 
```

Comment:
nit: indentations looks a little bit off here?

Reply:
@hanm for the general indention after { we use 4 white spaces, but for the statement breaking we usually use 8 white spaces, isn't that true for ZK codebase, I saw the previous code is using this rule.

Let me know if this is not true, I can update those.